src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+class Ball {
```

Comment:
lets override method `toString()`
this method is invoked by java when you pass this object to System.out.println() method